README.md

```diff
@@ -2,62 +2,160 @@
+
+```bash
+$ whoami
+```
+
+ < probably need screen shot>
+
+The `whoami` command lets you see which user account you're logged in to from a
+terminal window. This might seem obvious, especially if you're logged in on your
+personal computer, but it's not always clear which account you're you're working
+in the command line.
+
+In a strange circumstance where `whoami` isn't installed, there is another command
+you can use that can tell you your current username. Try:
+
+```bash
+$ id -un
+```
 
 ### Identifying the "Present Working Directory" With `pwd`
 
-Open up command prompt or terminal. Type in: `pwd` and hit return.
+With your Terminal program open, type in `pwd` and hit return/enter.
 
 ```bash
 $ pwd
 ```
 
 You should see some output describing the directory you are currently within.
+The `pwd` command stands for "**p**rint **w**orking **d**irectory".
+You'll see that the operating system created a directory under the "User"
+directory that was named the same as your `whoami` result
 
-`/Users/[your user name]]`
+`/Users/[your user name]`
 
-That output is describing a location on your computer. You have a file system and within that
-file system are directories and files.
+We would call this directory your "home" directory.
 
-The `pwd` command stands for "**p**rint **w**orking **d**irectory".
+That output is describing a location on your computer.
+
+`/User/kellyegreene` means that I am currently working within a directory `/Users`
+on the root of my machine, and then within that directory, a directory named
+`kellyegreene`. You might wonder what directory `Users` is in. Read on to
+learn more about the `root` directory which contains all directories....
+
+<something like that>
 
-`/User/kellyegreene` means that I am currently working within a directory `/Users` on the root
-of my machine, and then within that directory, a directory named `kellyegreene`.
+That's my home directory.
 
-That's my home directory. It belongs to the user I am currently logged in as. The placeholder for
-a user's home directory is the `~` ("tilde") symbol.
+Because it's so common to read files from, or move things into our home
+directories, `bash` lets us type `~` as a shortcut. We'll be working with this
+shortcut later on.
 
 Try this:
 
-**Note:** *Any time you see the* `$` *character, you shouldn't type it in. This is just a standard
-way to represent a bash prompt. Yours may or may not be a* `$`.
+**Note:** *Any time you see the* `$` *character, you shouldn't type it in.
+This is just a standard way to represent a bash prompt. Yours may or may not
+be a* `$`.
 
 ```bash
 $ cd ..
 $ pwd
 ```
 
-You should now see that you are one directory above where you were, in you're in a new terminal,
-you might see:
+You should now see that you are one directory above where you were, in you're
+in a new terminal, you might see:
 
 `/Users`
 
+You should be able to make a guess about what `cd` does, but we'll explain it
```

Comment:
Before this blurb: It's like the last part of the `pwd` fell off.

You "moved up" one level of nesting.